src/Core/IDecompiler.cs

```diff
@@ -29,6 +29,8 @@ namespace Reko.Core
     /// </summary>
     public interface IDecompiler
     {
+        public event EventHandler? ProjectChanged;
```

Comment:
Please add a doc comment explaining when this event is raised.

Reply:
Done 5d33dde